lib/helper.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+*
+* @package phpBB3
```

Comment:
Almost a verbatim copy. I moved from procedural driven to a static class, tweaked the names (removed the phpbb_ prefix mostly), and cleaned up the code a little bit. I didn't really touch the logic however as I didn't feel like trying to parse through what they were doing .. and I wanted to make sure there was feature parity with password hashing. I figure the code can be more optimized at a later date - for now it's working. ;-)